test/test_base_providers.py

```diff
@@ -133,15 +142,37 @@ def test_get_benchmark(self):
 
         assert_pint_frame_equal(self, benchmarks, expected_data)
 
+    def test_get_benchmark_scope_matters(self):
+        '''
+        Simple sanity test, to verify, that getting intensity benchmarks
+        takes in account primary scope - S1S2 or S3
+        '''
+        # benchmarks for default scope S1S2
+        bm_s1s2 = self.base_EI_bm.get_SDA_intensity_benchmarks(self.company_info_at_base_year)
```

Comment:
There's a lot hidden in this one line (not your fault)!  But here's the deal: for a given thing being produced, and a given benchmark, there needs to be a concept as to what is the intensity of that thing being produced.

In the case of Autos, the thing being produced is the passenger km (pkm), which is an S3 emission only.  It is counter-intuitive to me, but the Autos benchmark gives a pass to S1 and S2 emissions of vehicular production.  So the SDA intensity for autos needs to look at g CO2/pkm and how that aligns to the benchmark.  For Electricity, it's t CO2/MWh (or t CO2/GJ).  And that's principally S1+S2 (though because S2 is minimal, it's really S1).

So what's really needed is each benchmark to provide a proper intensity metric that applies appropriately across (or to) the scopes.

Reply:
You are commenting in the test case, not the production code - did you mean that the test case should be constructed differently?
The test case is designed to sanity check, that loading different EI benchmark results into different output of the function under test.
I couldn't test that proper intensity metrics are properly applied, because I don't know know what is the expected inputs and the expected outputs are, for the function under test.

Reply:
Fair enough.  And I think I've done a better job clarifying the issue in the long comment about properly computing EI based on scopes above.  Once that's handled, the approach to constructing and evaluating the test case becomes clear.

Reply:
in the latest commit, i adjusted tests for temperature score, to ensure, that weighted score is calculated as expected